tests/conformance/conformance.go

```diff
@@ -10,7 +10,9 @@ import (
 var containerTag = ""
 
 func done(files []string) {
-	err := os.WriteFile("/tmp/sonobuoy/results/done", []byte(strings.Join(files, "\n")), 0666)
+	const writeFilePerms = 0o666
+	//nolint:gosec
+	err := os.WriteFile("/tmp/sonobuoy/results/done", []byte(strings.Join(files, "\n")), writeFilePerms)
```

Comment:
Can we get rid of the `nolint` comment by lowering the permission here to `644`?

Reply:
I had tried that but then it complained about writing to a tempfile so I decided to go with the nolint as we are not really worried about the security around the tests.

Reply:
Did you try `0o600`?